package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "prepack": "yarn build-report --standalone --flow --esm",
     "build-all": "npm-run-posix-or-windows build-all:task",
-    "build-all:task": "yarn build-report && yarn build-cdt-lib && yarn build-devtools && concurrently 'yarn build-extension' 'yarn build-lr' 'yarn build-viewer' 'yarn build-treemap' 'yarn build-smokehouse-bundle' && yarn build-pack",
+    "build-all:task": "yarn build-report && yarn build-cdt-lib && yarn build-devtools && yarn build-types && concurrently 'yarn build-extension' 'yarn build-lr' 'yarn build-viewer' 'yarn build-treemap' 'yarn build-smokehouse-bundle' && yarn build-pack",
```

Comment:
Maybe this is where "build all" is ambiguous, but I use it 
when I mean "build all our things", but I wouldn't want it to include all the generated type files copied alongside source files. Should it be part of `prepack` instead? I don't think there's any reason local development would ever want them in place, you really just need them there so relative paths work when Lighthouse is a dependency.